frontend/cypress/tests/chartView/chartView_blurbs.cy.js

```diff
@@ -0,0 +1,12 @@
+describe('blurbs', () => {
+  it('shows blurbs', () => {
+    cy.get('.markdown.blurb')
+      .first()
+      .should('contain', 'first blurb');
+  });
+
+  it('has the correct number of valid blurbs', () => {
```

Comment:
> Yes, right now, we are only checking for plain text. We do not need to test if every markdown formatting feature works, but let us add Cypress tests to check one markdown formatted blurb (and maybe one HTML formatted blurb) to ensure it is being parsed instead of shown as is. @sopa301

Based on the previous comment, can we make the 2 blurbs use markdown and HTML formatting? Thanks!

Reply:
Okay, done!